google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/appengine/cloud/AppEngineToolsMenuAction.java

```diff
@@ -52,8 +55,12 @@ public void actionPerformed(AnActionEvent e) {
       List<RemoteServer<AppEngineServerConfiguration>> servers =
           RemoteServersManager.getInstance().getServers(serverType);
 
-      DeploymentConfigurationManager.getInstance(project).
-          createAndRunConfiguration(serverType, ContainerUtil.getFirstItem(servers));
+      try {
+        DeploymentConfigurationManager.getInstance(project).
+            createAndRunConfiguration(serverType, ContainerUtil.getFirstItem(servers));
+      } catch (Exception ex) {
```

Comment:
Can you catch NPE and then in the catch block include a comment pointer to the issue?